youtube_dl/extractor/wrzuta_playlist.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class WrzutaPlaylistIE(InfoExtractor):
+    """
+        this class covers extraction of wrzuta playlist entries
+        the extraction process bases on following steps:
+        * collect information of playlist size
+        * download all entries provided on
+          the playlist webpage (the playlist is split
+          on two pages: first directly reached from webpage
+          second: downloaded on demand by ajax call and rendered
+          using the ajax call response)
+        * in case size of extracted entries not reached total number of entries
+          use the ajax call to collect the remaining entries
+    """
+
+    IE_NAME = 'wrzuta.pl - playlist'
+
+    _VALID_URL = r'https?://(?P<uploader>[0-9a-zA-Z]+)\.wrzuta\.pl/playlista/' \
+                 '(?P<id>[0-9a-zA-Z]+)/(?P<playlist_title>[^\?\/]+)'
+
+    _TESTS = [{
+        'url': 'http://miromak71.wrzuta.pl/playlista/7XfO4vE84iR/moja_muza',
+        'playlist_mincount': 14,
+        'info_dict': {
+            'id': '7XfO4vE84iR',
+            'title': 'moja_muza',
+            'description': 'moja_muza',
+        },
+    }, {
+        'url': 'http://noname925.wrzuta.pl/playlista/0uSEHVfbJl2/top_electro_od_noname925',
+        'playlist_mincount': 34,
+        'info_dict': {
+            'id': '0uSEHVfbJl2',
+            'title': 'top_electro_od_noname925',
+            'description': 'top_electro_od_noname925',
+        },
+    }, {
+        'url': 'http://panczarek.wrzuta.pl/playlista/52K3cZacAe3/playlista_1',
+        'playlist_mincount': 31,
+        'info_dict': {
+            'id': '52K3cZacAe3',
+            'title': 'playlista_1',
+            'description': 'playlista_1',
+        },
+    }, {
+        'url': 'http://heroesf70.wrzuta.pl/playlista/6Nj3wQHx756/lipiec_-_lato_2015_muzyka_swiata',
+        'playlist_mincount': 144,
+        'info_dict': {
+            'id': '6Nj3wQHx756',
+            'title': 'lipiec_-_lato_2015_muzyka_swiata',
+            'description': 'lipiec_-_lato_2015_muzyka_swiata',
+        },
+    }, {
+        'url': 'http://andziaoox.wrzuta.pl/playlista/1TOgrSX1hFR/andziaoox_moja_playlista_1',
+        'playlist_mincount': 29,
+        'info_dict': {
+            'id': '1TOgrSX1hFR',
+            'title': 'andziaoox_moja_playlista_1',
+            'description': 'andziaoox_moja_playlista_1',
+        },
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        extracted_id = mobj.group('id')
+        uploader = mobj.group('uploader')
+        media_title = mobj.group('playlist_title')
```

Comment:
Title should be human readable. Take http://andziaoox.wrzuta.pl/playlista/1TOgrSX1hFR/andziaoox_moja_playlista_1 for example, the title can be ```andziaoox / moja playlista #1``` or ```moja playlista #1```.